docusaurus/docs/manage/02-router/01-overview.md

```diff
@@ -0,0 +1,63 @@
+---
+id: router-deploying-overview
+title: Deployment Overview
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import styles from './styles.module.css';
+
+In this article, we are describing the process of router deployment and what attributes can be updated, removed, added after the deployment is completed.  

```

Comment:
In technical docs, it's better to maintain a more formal tone and avoid informal words like "you", "we", "I" etc. Those sorts of words are better for a blog or tutorial type of doc. This sentence should be reworded:

**From:**
In this article, we are describing the process of router deployment and what attributes can be updated, removed, added after the deployment is completed.

**To:**
This article describes the process of deploying a router. It covers what attributes can be updated, removed and added after the deployment is completed.

Reply:
makes sense, I always the debate with my self how I should write this as I, we, or it